client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.scss

```diff
@@ -23,3 +26,34 @@
 .professional-email-card__discount.badge {
 	margin-bottom: 12px;
 }
+
+.email-provider-stacked-card__provider-form {
+	.titan-new-mailbox-list__main {
+		max-width: 100%;
+
+		@include break-mobile {
+			max-width: 440px;
+		}
+	}
+}
+
+.email-provider-stacked-card__provider-form > * .form-text-input-with-affixes {
+	flex-direction: column;
+
+	.form-text-input-with-affixes__suffix {
+		border-top-width: 0;
+		border-left-width: 1px;
+		border-left-style: solid;
+		border-left-color: var( --color-neutral-10 );
+	}
+
+	@include break-small {
+		flex-direction: row;
+		.form-text-input-with-affixes__suffix {
+			border-left-width: 0;
+			border-top-width: 1px;
+			border-top-style: solid;
+			border-left-color: var( --color-neutral-10 );
```

Comment:
Is there a reason these need to be re-declared for the larger size? I would prefer a more specific selector that always applies if that's needed.